front/components/navigation/NavigationSidebar.tsx

```diff
@@ -151,7 +154,13 @@ export const NavigationSidebar = React.forwardRef<
                                       label={nav.label}
                                       icon={nav.icon}
                                       className="TEST grow pl-14 pr-4"
-                                      href={nav.href}
+                                      link={
+                                        menu.href
```

Comment:
should be `nav` here (or rename to subMenu ?)

Reply:
Fixed.